src/pages/settings/Payments/PaymentMethodList.js

```diff
@@ -87,7 +87,7 @@ class PaymentMethodList extends Component {
                         bankAccount.accountNumber.slice(-4)
                     }`
                     : null;
-                const {icon, iconSize} = getBankIcon(bankAccount.additionalData.bankName);
+                const {icon, iconSize} = getBankIcon(_.get(bankAccount, 'additionalData.bankName', ''));
```

Comment:
Per the [style guide](https://github.com/Expensify/App/blob/main/STYLE.md#esnext-are-we-allowed-to-use-insert-new-language-feature-why-or-why-not) can you use `lodash/get` here instead of underscore?

Reply:
Done!